index.ts

```diff
@@ -158,6 +160,7 @@ const LOG = {
   PULLING: 'Pulling files...',
   PUSH_SUCCESS: (numFiles: number) => `Pushed ${numFiles} ${pluralize('files', numFiles)}.`,
   PUSH_FAILURE: 'Push failed. Errors:',
+  NO_NETWORK: 'Error: Looks like you are not connected to the internet',
```

Comment:
- Please put this message under `ERROR`
- `NO_NETWORK` -> `OFFLINE`
- Something like`Error: Looks like you are offline.`
  - Period to end sentence
  - Be terse